.github/workflows/posts-scheduler.yml

```diff
@@ -0,0 +1,40 @@
+name: Publish posts scheduled for today
+
+on:
+  schedule:
+    # Run every day at 00:00 UTC
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+
+jobs:
+  publish_posts:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout main branch
+        uses: actions/checkout@v4
+        with:
+          ref: main
```

Comment:
Can you change this for `master`? In al-folio our main branch is still named master, at least until #2086 is closed.